### PR TITLE
Dedup buildConfig

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
This is to abstract a common function to build kube rest config

### CODE REVIEW

File: pkg/kube/config.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Vulcan Authors.
```
Comment (k82cn) [inline pkg/kube/config.go]: s/Vulcan/Volcano

Comment (hzxuzhonghu) [reply pkg/kube/config.go]: Good catch, i replaced all

**Pull request merged**